embedding/csharp/Tizen.Flutter.Embedding/Channels/StandardMessageHelper.cs

```diff
@@ -60,7 +60,7 @@ public static void WriteSize(Stream stream, int size)
             {
                 stream.WriteByte(Convert.ToByte(size));
             }
-            else if (size <= 0xffff)
+            else if (size <= Int16.MaxValue)
```

Comment:
The size itself seems correct and consistent with the other language ([C++](https://github.com/flutter/engine/blob/0c57a50810e8f2e0f278722db9e28fae67adb4f5/shell/platform/common/client_wrapper/standard_codec.cc#L246-L251)) implementations. I think the `Convert.ToInt16` part should be changed to `Convert.ToUInt16` to match the C++ implementation (same for `Convert.ToInt32`).

Reply:
Will check how it works and fix it soon!

Reply:
Hi @swift-kim, should I add `Convert.ToUInt32(size)` also?

Reply:
Yes.

Reply:
@swift-kim updated! Could you please merge my fix? (If no other remarks)

Reply:
Hmm. What I meant by "the two lines" were actually lines 66 and 88, but let me fix this later.

Reply:
@swift-kim sorry for misunderstanding